services/patient.services.js

```diff
@@ -35,6 +35,41 @@ const createPatient = async ({
   }
 };
 
+const editPatient = async ({
+  id,
+  image,
+  firstName,
+  lastName,
+  dateOfBirth,
+  address,
+  city,
+  phoneNumber,
+  email
+}) => {
+  if ((await db.Patient.findOne({ where: { id } })) === null) {
+    throw new MedicaError("Patient does not exists");
+  }
+  try {
+    const patient = await db.Patient.findOne(
+      { where: { id } }
+    );
+    patient.set({
```

Comment:
No need for this. You can simply pass object and then:

```
patient.set(data)
```

You have validated data so no need to destruct (at least for now)